pandas/tests/arithmetic/test_numeric.py

```diff
@@ -150,14 +150,6 @@ def test_numeric_arr_mul_tdscalar(self, scalar_td, numeric_idx, box):
     def test_numeric_arr_rdiv_tdscalar(self, three_days, numeric_idx, box):
         index = numeric_idx[1:3]
 
-        broken = (isinstance(three_days, np.timedelta64) and
-                  three_days.dtype != 'm8[ns]')
```

Comment:
we actually had this in a test ...hahha

Reply:
Yah, this is to xfail the broken stuff this PR fixes